### ISSUE TITLE
Loss curves

### ISSUE THREAD

**Issue by TropComplique** [HyeongseokSon1/SRFeat#16]
Hi!
Could you please release your training loss curves for the GAN model ?
I am talking about summaries files generated by `main_gan_train.py`.

I am trying to reproduce your paper in pytorch.
Right now I am having problems because the discriminators are too powerful:
their loss goes to zero quite fast.
